CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 ## Improvements
 
 - [connectivity] Add details to error message for missing "URL" properties in destinations.
+- [connectivity] Use the cache of `xssec` library to cache verification certificates.
```

Comment:
I guess you need to merge the main branch, as other changelog items were released.